workbench-core/base/src/aws/helpers/s3Service.ts

```diff
@@ -89,15 +89,8 @@ export default class S3Service {
     prefix: string,
     timeToLiveSeconds: number
   ): Promise<string> {
-    const config: S3ClientConfig = {
```

Comment:
are you sure this isn't something SWB needs?

Reply:
Replacing the above with `this._s3` does the same thing. It also includes the rest of the config without having to explicitly add it.

Reply:
sounds good to me!